dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/FieldAccessNode.java

```diff
@@ -118,4 +128,13 @@ public int hashCode() {
     public Collection<Node> getOperands() {
         return Collections.singletonList(receiver);
     }
+
+    /**
+     * Retrieves the modifiers of the method.
+     *
+     * @return the set of modifiers of the method
+     */
+    public Set<Modifier> getMethodModifiers() {
```

Comment:
Is this method used somewhere? If not, I would remove it.

Reply:
ok, I will remove it.